fibers.texi

```diff
@@ -83,14 +83,17 @@ One of the most dominant patterns for concurrency these days is
 Node.js run-time for JavaScript.  The basic observation in the
 callback approach to concurrency is that the efficient way to handle
 tens of thousands of connections at once is with low-level operating
-system facilities like @code{poll} or @code{epoll}.  You add all of
-the file descriptors that you are interested in to a ``poll set'' and
-then ask the operating system which ones are readable or writable, as
-appropriate.  Once the operating system says ``yes, file descriptor
-7145 is readable'', you can do something with that socket; but what?
-With callbacks, the answer is ``call a user-supplied closure'': a
-callback, representing the continuation of the computation on that
-socket.
+system facilities like @code{poll}, @code{epoll} or
+@code{kqueue}. Fibers will use the native events system facility
+implementation when avaliable (e.g. @code{epoll} on Linux systems) but
```

Comment:
Typo: avaliable -> available.

Also, this appears to be untrue, going by configure.ac, when libevent is requested, that's what i sused, even if epoll is available.

Reply:
I've changed that to

```
By default, Fibers will try to use the native events system facility
implementation (e.g. @code{epoll} on Linux systems) but
@code{libevent} is also supported for portability reasons
```

I believe this is true.